n4m-vimeo/README.md

```diff
@@ -0,0 +1,20 @@
+# n4m-vimeo
```

Comment:
I would not create an entirely separate readme for node. Let's conslidate this all into the root readme... which will save you some work on having to rewrite the same things twice.